.eslintrc.js

```diff
@@ -15,6 +15,12 @@ module.exports = {
   overrides: [
     {
       files: ["tests/**/*.js"],
+      rules: {
+        "no-console": 0,
+      },
+    },
+    {
+      files: ["tests/a11y/**/*.js", "tests/e2e/**/*.js"],
```

Comment:
This only applies to the code in the proxy server. Whatever arguments might exist against `console.log`, they require way too much complexity for our very simple needs here. 😛